pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -157,6 +168,14 @@ func fixLibvirtEnabled() (bool, error) {
 }
 
 func checkUserPartOfLibvirtGroup() (bool, error) {
+	return checkUserPartOfGroup("libvirt")
+}
+
+func checkUserPartOfLibvirtdGroup() (bool, error) {
+	return checkUserPartOfGroup("libvirtd")
```

Comment:
For the record, see https://github.com/code-ready/crc/issues/622 , the `libvirtd` group is not needed, `libvirt` group is fine on recent ubuntus.